docs/detections/api/exceptions/lists-index-api-overview.asciidoc

```diff
@@ -1,17 +1,17 @@
 [[lists-index-api-overview]]
-=== Lists index endpoint
+=== Lists data stream(/index) endpoint
```

Comment:
so, the endpoint is `api/lists/index`, but we create data stream. So, I put `/index` in parentheses here. Let me know, if there any better way to express it